### DISCUSSION TITLE
[Feture Request] Add assembly view

### DISCUSSION THREAD

**Discussion by lixin-wei** [microsoft/vscode-cpptools#7325] [Ideas]
in gdb, there is a command `layout asm` to switch to assembly view of the running codes. And we can debug instruction by instruction.

This is very useful when some part of my code is written in assembly language.

Does VSCode CPP plugin plan to support a similar feature? i.e. :
1.  switch views between source code and assembly instruction
2. debug instruction by instruction

![image](https://user-images.githubusercontent.com/19600697/114137265-80a12a80-993e-11eb-9f7f-ee2d444645ad.png)

**Comment by michelleangela** [comment]
@lixin-wei 
We're currently working on this feature, see issue https://github.com/microsoft/vscode-cpptools/issues/206.

**Reply by lixin-wei** [reply]
Oh, that's great!
